### ISSUE TITLE
Add a logout button for platforms

### ISSUE THREAD

**Issue by artandor** [artandor/multistream-tools#25]
As a USER,
I want to be able unlink a specific Account

It would allow to link another from same platform, aswell as allow users to remove their data if wanted.

Todo:

- [ ] Add a logout button if the user linked the platform
- [ ] The logout button should remove the account from database

**Comment by vasilvestre** [comment]
Kind of duplicate of https://github.com/artandor/multistream-tools/issues/16, I think this issue overseed it

**Comment by artandor** [comment]
Mmmh good question. I believe we should forbid unlinking if the user only has one account linked ?

I see what you edited, is the issue clear now ?

**Comment by vasilvestre** [comment]
I think we could let people unlink all platform except one until users ask for change

**Comment by artandor** [comment]
Yeah that's what i meant earlier. Being able to unlink all except the last linked account seems good.

And if he wants to delete the last one, we tell him to go to another part "delete account" where we delete his User ? (Another issue)
